msteams-platform/tabs/how-to/authentication/auth-oauth-provider.md

```diff
@@ -84,11 +84,11 @@ The following image provides the flow to add authentication to external browsers
    |`authId` | The request-id Teams created for this specific authentication request that needs to be sent back to Teams through deeplink.|
 
     > [!TIP]
-    > The 3P app can marshal `authId`, `oauthRedirectMethod` in the OAuth `state` query parameter while generating the login URL for the OAuthProvider. The `state` contains the passed `authId` and `oauthRedirectMethod`, when OAuthProvider redirects back to the 3P server and the 3P app uses the values for sending authentication response back to Teams as described in **6. The 3P app server response to Teams**.
+    > The app can marshal `authId`, `oauthRedirectMethod` in the OAuth `state` query parameter while generating the login URL for the OAuthProvider. The `state` contains the passed `authId` and `oauthRedirectMethod`, when OAuthProvider redirects back to the server and the app uses the values for sending authentication response back to Teams as described in **6. The 3P app server response to Teams**.
```

Comment:
@v-rmvv thanks for the suggestion. though this may boost the arcolinx score, using third-party from 3P does stray away from Microsoft terminology we have used in other docs.

Reply:
@mobajemu-msft this can be resolved, right? I see updates to use "third-party" in the changes.